client/src/charts/CircleProportionGraph/CircleProportionGraph.js

```diff
@@ -0,0 +1,160 @@
+import { scaleOrdinal } from "d3-scale";
+import _ from "lodash";
+import React, { Fragment } from "react";
+
+import { DisplayTable, Format } from "src/components/index";
+
+import { newIBCategoryColors } from "src/core/color_schemes";
+
+import { TabularLegend } from "src/charts/legends";
+
+import {
+  InteractiveGraph,
+  create_text_maker_component_with_nivo_common,
+  general_default_props,
+  get_formatter,
+} from "src/charts/wrapped_nivo/wrapped_nivo_common";
+
+import text from "./CircleProportionGraph.yaml";
+
+import "./CircleProportionGraph.scss";
+
+const { text_maker, TM } = create_text_maker_component_with_nivo_common(text);
+
+export class CircleProportionGraph extends React.Component {
+  render() {
+    const {
+      is_money,
+      formatter,
+      height,
+      child_value,
+      child_name,
+      parent_value,
+      parent_name,
+      disable_table_view,
+      table_name,
+    } = this.props;
+
+    const color_scale = scaleOrdinal().range(newIBCategoryColors);
+    const value_formatter = get_formatter(is_money, formatter, true, false);
+
+    const Circles = () => {
+      const viewbox_dimension = 180;
+      const parent_radius = viewbox_dimension / 2;
+      const parent_cx = viewbox_dimension / 2;
+      const parent_cy = viewbox_dimension / 2;
+
+      const child_radius = _.max([
+        parent_radius * Math.sqrt(child_value / parent_value),
+        viewbox_dimension * 0.01, // roughly the smallest visible child radius
+      ]);
```

Comment:
<img width="1680" alt="Screen Shot 2021-08-17 at 10 23 39 AM" src="https://user-images.githubusercontent.com/60910369/129744173-f9a8e8d0-c5a8-4d31-9312-0bf2f13bff62.png">
I think 0.01 * viewbox_dimension is an appropriate value as the minimum child radius.